collect_app/src/main/java/org/odk/collect/android/audio/AudioPlayerViewModel.java

```diff
@@ -178,16 +178,14 @@ public void errorDisplayed() {
     }
 
     private void schedulePositionUpdates() {
-        if (positionUpdatesCancellable == null) {
-            positionUpdatesCancellable = scheduler.repeat(() -> {
-                CurrentlyPlaying currentlyPlaying = this.currentlyPlaying.getValue();
+        positionUpdatesCancellable = scheduler.repeat(() -> {
```

Comment:
We want a test for this change.

Reply:
I verified `playMultipleClips_updatesProgress_forAllClips()` and `playInOrder_playsClipsOneAfterTheOther_andUpdatesProgress` fails on master.